test/example.test.js

```diff
@@ -1,16 +1,30 @@
 // IMPORT MODULES under test here:
-// import { example } from '../example.js';
+import { countsAsAYes } from '../utils.js';
 
 const test = QUnit.test;
 
-test('time to test a function', (expect) => {
+test('test should return a true statement when the given a word that starts with y', (expect) => {
     //Arrange
     // Set up your arguments and expectations
     const expected = true;
-    
+
+    //Act 
+    // Call the function you're testing and set the result to a const
+    const actual = countsAsAYes('yes');
+
+    //Expect
+    // Make assertions about what is expected versus the actual result
+    expect.equal(actual, expected);
+});
+
+test('test should return a true statement when the given a word that starts with y', (expect) => {
```

Comment:
Just needs to be changed to "should return a FALSE statement when given a word that DOES NOT start with a y"